webapp-django/crashstats/home/static/home/js/home.js

```diff
@@ -31,18 +31,10 @@ $(function () {
 
     var COLORS = ['#6a3d9a', '#e31a1c', '#008800', '#1f78b4'];
 
-    var container = $('#homepage-graph-container');
-    var superSearchApiUrl = container.data('supersearch-api');
-    var productBuildTypesApiUrl = container.data('product-build-types-api');
-    var adiApiUrl = container.data('adi-api');
+    var datapanel = $('#homepage-data-panel');
 
-    var product = container.data('product');
-    var versions = container.data('versions');
-    var platforms = container.data('platforms');
-    var duration = container.data('duration');
-    var esShardsPerIndex = container.data('es-shards-per-index');
-
-    var pageTitle = $('title').text();
+    var product = datapanel.data('product');
+    var versions = datapanel.data('versions');
```

Comment:
`product` is unused in the rest of the remaining file, and is in a closure, so other scripts can't use it either.

`versions` is only used to generate `betaVersions`, which is in a similar state to `product`.

Looking through the whole file, I think the only things that are still used in the entire file are the bits that set the colors for the version headers, and you could get rid of that completely with CSS or the [cycle](https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#cycle) builtin. Seems like it's worth doing now that this page is so simple.

Reply:
So... it sounds like you're saying "this works, but it could be cleaner code". Is that right?

Assuming that's right, I'm not sure I understand exactly what you're asking for, so it'd take me some modicum of time to figure it out and do it. I think I'd rather land this now because it's blocking other things I need to get done and the JS/CSS things will get cleaned up during Photonic or some followup PR.

Reply:
Scratch that. I feel lousy after writing that comment. I'll try to fix it tonight and we can push forward tomorrow.

Reply:
I'm not sure whether the colors are important or not. Previously it was used to make sure the header color was the same as the color used in the graph line. We don't have graphs now. Maybe the header colors don't matter. I think I'll just nix them and we can re-add them later if need be.